decorators/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0...9]
+  end
+end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please correct your `TrimmerDecorator#correct_name` method as it doesn't return 10 characters as required:

![image](https://github.com/mudasiri/rubyschool-library/assets/77069011/f3301f13-8cfa-4f4c-84d5-6e70e2ae86e6)

This happens because the three dots `...` refer to a non-inclusive range, meaning that the char at position `9` is not included. Please make sure to correct this! 🤓 🔧 


```suggestion
class TrimmerDecorator < BaseDecorator
  def correct_name
    @nameable.correct_name[0..9]
  end
end
```